getnacelle/nacelle-js
# chore: Shopify Storefront API version 2022-10

## Why are these changes introduced?

Addesses [ENG-7394](https://nacelle.atlassian.net/browse/ENG-7394)

> Update to use 2022-10 version of the Shopify Storefront API

## What is this pull request doing?

- Updates `@nacelle/shopify-cart` to use the latest (`2022-10`) version of the Shopify Storefront API.
- Updates the lockfile for the Next.js Reference Store. This happened automatically when running `npm run dev` in that project (`next` detects lockfile issues and corrects them automatically).

## How to test

1. Check out the `ENG-7394-shopify-storefront-api-version-2022-10` branch.
2. `npm run bootstrap` in the monorepo root.
3. Navigate to `packages/shopify-cart`.
4. `npm run build` - the build should produce no errors.
5. `npm run test:ci` - all tests should pass with 100% coverage.

## Local testing results

In the Next.js Reference Store, the cart functioned as expected using the locally-bootstrapped `@nacelle/shopify-cart`:

https://user-images.githubusercontent.com/5732000/197248133-7a3cd3ab-bcc1-41e1-8a30-17569943846b.mov
